src/init.cpp

```diff
@@ -1149,7 +1125,7 @@ bool AppInitSanityChecks()
     init::SetGlobals();
 
     // Sanity check
-    if (!InitSanityCheck())
+    if (!init::SanityChecks())
```

Comment:
e01dfceb3c2f355e8b9979eb6aa6c41550c37787: Could add `{` while touching this line?